STARTERKIT/js/script.js

```diff
@@ -10,15 +10,32 @@
 // wrapping it with an "anonymous closure". See:
 // - https://drupal.org/node/1446420
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
-(function (Drupal) {
+(function (Drupal, $, debounce, drupalSettings) {
 
   'use strict';
 
   // To understand behaviors, see https://drupal.org/node/756722#behaviors
-  Drupal.behaviors.my_custom_behavior = {
+  Drupal.behaviors.slider = {
```

Comment:
i suggest to create a new file for sliders. Not script.js

Reply:
Agree, thank you for review.